man/timesyncd.conf.xml

```diff
@@ -103,6 +103,17 @@
         Defaults to 30 seconds and must not be smaller than 1 second.</para></listitem>
       </varlistentry>
 
+      <varlistentry>
+        <term><varname>SaveIntervalSec=</varname></term>
+        <listitem><para>Interval to periodically save current time. This is especially useful for offline
```

Comment:
Suggestion: "The interval at which the current time is periodically saved to disk, in the absence of any recent synchronisation from a server."

Please also update the systemd-timesyncd.service man page which has some documentation that describes the updating of the `/var/lib/systemd/timesync/clock` file.